### PR TITLE
[Kogito-8799] Add Kubernetes security profile to Kaniko and builder pods

### PR DESCRIPTION
**Pull request by desmax74**
**Description of the change:**

**Motivation for the change:**

**Checklist**

- [ ] Add or Modify a unit test for your change
- [ ] Have you verified that tall the tests are passing?

### CODE REVIEW

File: util/kubernetes/container.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package kubernetes
```
Comment (spolti) [inline util/kubernetes/container.go:1]: rename to security_context.go?

File: util/kubernetes/container.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import (
+	"github.com/kiegroup/container-builder/util"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// This is the security defualt to run without privileges
```
Comment (MarianMacik) [inline util/kubernetes/container.go:8]:
```suggestion
// This is the security default to run without privileges
```

File: util/kubernetes/pod.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package kubernetes
+
+import (
+	"github.com/kiegroup/container-builder/util"
+	corev1 "k8s.io/api/core/v1"
+)
+
```
Comment (MarianMacik) [inline util/kubernetes/pod.go:7]: Maybe add documentation here as well?

File: util/kubernetes/container.go (hunk 1 above)
Comment (MarianMacik) [reply util/kubernetes/container.go:1]: + I would merge 2 files in one.

**Review by ricardozanini — COMMENTED** [review]
Looks good. We can merge as soon as other comments are addressed.

**Comment by ricardozanini** [comment]
Closing since we are moving to https://github.com/kiegroup/kogito-serverless-operator

**Pull request closed without merging**